LanguageExt.Core/Compose.cs

```diff
@@ -12,17 +12,65 @@ public static class __ComposeExt
         /// <summary>
         /// Function composition
         /// </summary>
-        /// <returns>b(a(v))</returns>
+        /// <returns>v => b(a(v))</returns>
         [Pure]
         public static Func<T1, T3> Compose<T1, T2, T3>(this Func<T2, T3> b, Func<T1, T2> a) =>
             v => b(a(v));
 
         /// <summary>
+        /// Function composition
+        /// </summary>
+        /// <returns>() => b(a())</returns>
+        [Pure]
+        public static Func<T2> Compose<T1, T2>(this Func<T1, T2> b, Func<T1> a) =>
+            () => b(a());
+
+        /// <summary>
+        /// Action composition
+        /// </summary>
+        /// <returns>v => b(a(v))</returns>
+        [Pure]
+        public static Func<T1, Unit> Compose<T1, T2>(this Action<T2> b, Func<T1, T2> a)
```

Comment:
Also, you don't need a separate overload for an argument of type `this Func<T2, Unit>`, that's already covered by the existing single argument function composition overloads.